src/analysis/retail/monk/windwalker/modules/spells/HeartOfTheJadeSerpent.tsx

```diff
@@ -101,14 +101,16 @@ class HeartOfTheJadeSerpent extends HotJS {
         )}
         {this.currentSotwl !== 0 && (
           <>
+            <br />
```

Comment:
note: generally should avoid `<br/>` for layout as it can produce awkward results on differing screen sizes. however, tooltips can be finicky. in the interest of timeliness i'm going to merge the APL changes, since this is super minor.